src/build/__main__.py

```diff
@@ -63,12 +63,9 @@ def _setup_cli() -> None:
 
     if platform.system() == 'Windows':
         # try to enable colors
-        try:
-            import colorama
+        import colorama
 
-            colorama.init()
-        except ModuleNotFoundError:
-            pass
+        colorama.init()
```

Comment:
I believe this was done on purpose to allow boostrapping build without colorama if so desired.

Reply:
Do you need it? Or who then?

Reply:
https://github.com/pypa/build/pull/300#issuecomment-846318593

Reply:
@FFY00 is anybody using `build` without `colorama` on Windows? I guess that needs a separate package with `colorama` commented out from dependencies. I removed import check, because otherwise it needs to be covered by tests.

Reply:
But they won't be able to use it without installing dependencies. So they need to checkout the source and make edits to `setup.cfg`. In the same way they could just comment `colorama`.

Maybe it is better to use `NO_COLOR` than rebuilding the package?

https://github.com/pypa/build/blob/4edd5effe9912eee825ba9532255166dadc08a25/src/build/__main__.py#L37

It looks like a better place for `colorama` import.

Reply:
They will, you can install the package without it's dependencies. The point is that it won't blow up at runtime if the dependency is missing.

The main use-case for this is bootstraping a Python environment, where `build` is one of the packages required to do so. See https://github.com/FFY00/python-bootstrap for an example.

> It looks like a better place for `colorama` import.

Not really :rofl: 

`_setup_cli` is meant to contain all code that changes external state, which you only want to happen when you actually run the CLI. This should be better documented, sorry.

Reply:
Ok. I get back that import check. Coverage will fail.

Reply:
No worries, Codecov is a bit annoying, even though the job will fail, it is not required for merging.

Reply:
I see. Got the import catch back.